src/components/container/ConnectDevice/ConnectDevice.tsx

```diff
@@ -76,7 +76,7 @@ export default function (props: ConnectDeviceProps) {
 	}
 
 	function connectToDevice() {
-		MyDataHelps.connectExternalAccount(props.providerIDCallback());
+		MyDataHelps.connectExternalAccount(props.providerIDCallback(), { openNewWindow: true });
```

Comment:
@bxh I realized an issue with this - we need to actually refresh the data for these various component when the connection happens...e.g. we should be doing 

```MyDataHelps.connectExternalAccount(props.providerIDCallback(), { openNewWindow: true }).then(function() {
///refresh component data
});
```

Reply:
Otherwise the connection completes and you are left back on the screen and nothing has changed